js/viz/docs/docBaseWidget.js

```diff
@@ -35,8 +35,9 @@ var BaseWidget = {
     /**
     * @pseudo CommonVizPalette
     * @type Array<string>|string
-    * @default "Default"
-    * @acceptValues "Default" | "Soft Pastel" | "Harmony Light" | "Pastel" | "Bright" | "Soft" | "Ocean" | "Vintage" | "Violet" | "Carmine" | "Dark Moon" | "Soft Blue" | "Dark Violet" | "Green Mist"
+    * @default "Material"
+    * @deprecatedAcceptValues "Default"
```

Comment:
@roman-simionov 
Move to the "Enums.VizPalette" enum declaration.

@lukyanovas 
How to mark the "Enums.VizPalette" enum item as deprecated?

Reply:
@San4es currently we have no special syntax for deprecating a single item, though it's possible to deprecate a whole enum instead.

Reply:
We've discussed this with @lukyanovas and suggest to use the following syntax:

``` js
/**
 * @typedef {string} Enums.VizPalette
 * @enum {'Bright'|'Carmine'|'Dark Moon'|'Dark Violet'|'Default'|'Green Mist'|'Harmony Light'|'Ocean'|'Pastel'|'Soft'|'Soft Blue'|'Soft Pastel'|'Vintage'|'Violet'}
 * @member 'Default' @deprecated 'Material'
 */
```
Now this will be ignored, but we'll support it in the nearest future.

@San4es, what do you think about?